### ISSUE TITLE
wpa_supplicant.conf question

### ISSUE THREAD

**Issue by ktsi** [EliasKotlyar/Xiaomi-Dafang-Hacks#144]
Hi all,

I have successfully installed the microSD bootloader. Then I copied the contents of firmware_mod folder to root of micro SD card and elite the wpa_supplicant.conf file.

My file is exactly as bellow:

ctrl_interface=/var/run/wpa_supplicant
ctrl_interface_group=0
ap_scan=1

network={
        ssid=“myssid”
        key_mgmt=WPA-PSK
        pairwise=CCMP TKIP
        group=CCMP TKIP WEP104 WEP40
        psk=“mypassword”
        priority=2
}

However I cannot see the camera in my local network. Is smh wrong with the wpa_supplicant.conf file?

All help appreciated.

Thanks

**Comment by helio58** [comment]
@ktsi
Hi I hade problems with that too.

Solvit changing:
pairwise=CCMP
group=CCMP

Using WPA-personal and AES on router.
Giv it a try.

**Comment by ktsi** [comment]
Hi, I tried your approach but still nothing. Let me ask, when the camera connects to the network what colour is the LED? Is it blue?

Thanks for your reply.

**Comment by helio58** [comment]
Blue is when the Mi home, with this Hack its orange. When you start the camera it gets blue for 5 seconds den orange. If it don't get blue during 5 seconds you haven't install the hack.

**Comment by ktsi** [comment]
When I start the camera (with SD card inserted) I get the blue light for about a second. However when I first installed the boot loader I saw the blue light for about 5 secs that’s why I am quite sure that the hack was done ok.

**Comment by helio58** [comment]
Seems right, 
if you remove the sd card the blue light should be on for 5 secs.
It should be something with the "wpa_supplicant.conf" and the router settings.

**Comment by ktsi** [comment]
When I boot the camera without SD card then the light goes yellow, then blue for about 5 seconds, then again yellow and then it turns off. I am confident that the bootloader is installed correctly. However when I have the micro SD card inside I cannot see the camera in my local network.
How should my access it? 

http:\\localip\chi-bin\status

Is that it?

**Comment by helio58** [comment]
http://Dafang_IP/cgi-bin/status.cgi

**Comment by ktsi** [comment]
Yeah, this is what I meant. Problem is camera does not connect. It is driving me crazy.

**Comment by FreakErn** [comment]
Maybe the camera does not even run the run.sh file, like mine does... ( #139 )
But i guess to find that out, you'll have to solder the pin header to connect a ftdi adapter

**Comment by ktsi** [comment]
How does one go back to original FW? After having flashed the boot loader I mean. Do I just remove the microSD card? #losingfaith

**Comment by helio58** [comment]
Mine did not. 
I had to put the original firmware into the SD card and flashed as you flash the hack.
The original firmware can be download at https://github.com/EliasKotlyar/Xiaomi-Dafang-Hacks/tree/master/firmware_original then rename it to demo.bin.

**Comment by rhass** [comment]
@ktsi Is your SSID hidden?

**Comment by ktsi** [comment]
Hi there. My SSID is not hidden. 
I really cannot understand what is happening.

Can someone please share his/her version of wpa_supplicant.conf file?

Thanks a lot in advance.

**Comment by rhass** [comment]
@ktsi Try adding `scan_ssid=1` to your `wpa_supplicant.conf`. If that doesn't work try adding the mac address of your wireless AP to an additional line `bssid=xx:xx:xx:xx:xx:xx`. If neither of those work help, then I suspect your wireless AP has some issues... I have had a couple of *really* terrible wireless routers and access points in the past myself where some devices couldn't connect at all to them. 

Good luck!
